### PR TITLE
ethclient: include withdrawals in ethclient block responses

### PR DESCRIPTION
**Pull request by peterbitfly**
This PR adds the Withdrawals array to the block body for blocks requested via the ethclient module.

### CODE REVIEW

File: ethclient/ethclient.go [hunk 1]
```diff
@@ -102,9 +102,10 @@ func (ec *Client) PeerCount(ctx context.Context) (uint64, error) {
 }
 
 type rpcBlock struct {
-	Hash         common.Hash      `json:"hash"`
-	Transactions []rpcTransaction `json:"transactions"`
-	UncleHashes  []common.Hash    `json:"uncles"`
+	Hash         common.Hash         `json:"hash"`
+	Transactions []rpcTransaction    `json:"transactions"`
+	UncleHashes  []common.Hash       `json:"uncles"`
+	Withdrawals  []*types.Withdrawal `json:"withdrawals"`
```
Comment (MariusVanDerWijden) [inline ethclient/ethclient.go:108]:
```suggestion
	Withdrawals  []*types.Withdrawal `json:"withdrawals,omitempty"`
```
I think omitting the withdrawals if empty is more in line with how we marshal other stuff to json

**Review by MariusVanDerWijden — APPROVED** [review]
LGTM

File: ethclient/ethclient.go (hunk 1 above)
Comment (peterbitfly) [reply ethclient/ethclient.go]: I was not sure if that object gets serialized to json at some point (as the library is mainly used for parsing the json rpc api responses into go types) but I've pushed a fix anyway. Thanks for the suggestion.

**Review by karalabe — APPROVED** [review]
SGTM

**Comment by karalabe** [comment]
@fjl Pls double check this tomorrow

**Pull request merged**